src/Collection.php

```diff
@@ -72,16 +72,24 @@ public function endpoint_callback( \WP_REST_Request $request ) {
 	protected function loop() {
 		$data = [];
 
+		$meta = [];
+
 		$this->query = new \WP_Query( $this->args );
 		while ( $this->query->have_posts() ) {
 			$this->query->the_post();
 			$data[] = $this->format_item( $this->query->post );
+
+			if ( empty( $meta ) ) {
```

Comment:
But we are using the first post as the meta for the the complete collection I don't think that's the correct way, since if this is going to be a blog collection why we are using the first element as the meta data for that.